katello/katello-backup

```diff
@@ -1,94 +1,83 @@
-#!/bin/bash
+#!/usr/bin/env ruby
 
-usage(){
-  echo "Usage: katello-backup <backup directory>"
-  echo " eg: $ katello-backup /tmp/katello-backup/"
-}
+require 'optparse'
+require 'fileutils'
 
-if [ "$#" -ne 1 ]; then
-  echo "ERROR: Please specify an export directory"
-  usage
-  exit 1
-fi
+optparse = OptionParser.new do |opts|
+  opts.banner = "Usage: katello-backup /path/to/dir [options]\n eg: $ katello-backup /tmp/katello-backup"
+end
 
-# Backup dir
-umask 0027
-BDIR=$1
-mkdir -p $BDIR
-chgrp postgres $BDIR
-chmod 770 $BDIR
-cd $BDIR
-echo "Creating backup folder $BDIR"
-echo ""
+optparse.parse!
 
-# Stop services
-katello-service stop
-service postgresql stop
+DIR = ARGV.pop
+if DIR.nil?
+  puts "**** ERROR: Please specify an export directory ****"
+  puts optparse
+  exit(-1)
+else
+  FileUtils.mkdir_p(DIR)
+  puts "Creating backup folder #{DIR}"
+  `chgrp postgres #{DIR}`
```

Comment:
since we're already using `FileUtils` for other things, any objection to changing this to `FileUtils.chown(nil, 'postgres', DIR)`?